esrally/track/params.py

```diff
@@ -692,7 +707,21 @@ def __iter__(self):
     def __next__(self):
         if self.conflicting_ids is not None:
             if self.conflict_probability and self.id_up_to > 0 and self.rand() <= self.conflict_probability:
-                doc_id = self.conflicting_ids[self.randint(0, self.id_up_to - 1)]
+                # a recency of zero means that we don't care about recency and just take a random number
+                # within the whole interval.
+                if self.recency == 0:
+                    idx = self.randint(0, self.id_up_to - 1)
+                else:
+                    # A recency > 0 biases id selection towards more recent ids. The recency parameter decides
+                    # by how much we bias. See docs for the resulting curve.
+                    #
+                    # idx_range is in the interval [0;1].
```

Comment:
I guess there's a typo here, `[0;1]` -> `[0,1]`

Reply:
It was actually intentional. The German decimal separator is a comma so it could be misinterpreted by a German as 1/10 instead of the interval from zero to 1 (inclusive). However, we can change it as well.

Reply:
I changed it anyway now as I've seen that we use the comma notation in all other places.